README.md

```diff
@@ -155,11 +155,13 @@ public class FingerprintApiExample {
         // Search events with custom filters
         try {
             // By visitorId
-            EventsGetResponse response = api.searchEvents(LIMIT, FPJS_VISITOR_ID, null, null, null, null, null, null, null);
+            EventsGetResponse response = api.searchEvents(LIMIT, null, FPJS_VISITOR_ID, null, null, null, null, null, null, null);
+            // Next page
+            // EventsGetResponse response = api.searchEvents(LIMIT, paginationKey, FPJS_VISITOR_ID, null, null, null, null, null, null, null);
```

Comment:
We could show here that `paginationKey` should be used from `response` as `response.paginationKey`